ovirt-imageio/main.go

```diff
@@ -0,0 +1,199 @@
+package main
```

Comment:
I forgot to merge https://github.com/kubev2v/ovirt-imageio-populator/pull/8 previously
so this file looks different now, as well as the CRD